recipes/libcurl/all/conanfile.py

```diff
@@ -1,4 +1,4 @@
-import os
+import os, platform
```

Comment:
```suggestion
import os
```
Shouldn't be necessary if you accept my other change.